onnx/defs/text/defs.cc

```diff
@@ -0,0 +1,104 @@
+#include "onnx/defs/schema.h"
```

Comment:
There should be a file `Operators-text.md` as there is a file `Operators-ml.md`. Maybe that can be done separately.

Reply:
Are you proposing a new domain for text-based operators out of curiosity?

Reply:
That's what my comments entails but you're right, I remember we only decided to move the text operators in a separate folder and not in a separate domain. Forget my comment. Maybe this question will come again if we keep adding more text operators.